repo/packages/M/msoms/0/package.json

```diff
@@ -6,7 +6,10 @@
  "maintainer": "omscontainers@microsoft.com",
  "description": "Microsoft Operations Management Suite(MS OMS) – Container Solution provides real-time analytics for operational data (Performance, Inventory) and monitoring for Docker Containers.",
  "framework": false,
- "tags": ["msoms"],
+ "tags": [
+   "msoms",
+   "Microsoft"
+ ],
  "licenses": [
```

Comment:
Can you add  some more description to 'preInstallNotes' on how to get the workspace key and ID?